docs/user-interface/images/app-icons.md

```diff
@@ -1,83 +1,215 @@
+The transformed name, the resource without the path or extension, is `healthapp`. The values for `android:icon` and `android:roundIcon` would be `@mipmap/healthapp` and `@mipmap/healthapp_round`, respectively. The android manifest should be updated to match `healthapp` as the icon:
 
-> [!IMPORTANT]
-> The `Color` attribute can still be specified when producing an adaptive launcher icon on Android so that a background color is generated for the app icon on other platforms.
+```xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <application android:allowBackup="true" android:icon="@mipmap/healthapp" android:roundIcon="@mipmap/healthapp_round" android:supportsRtl="true"></application>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.INTERNET" />
+</manifest>
+```
+
+> [!TIP]
+> Instead of creating new image files for the icon, simply replace the two image files provided by the .NET MAUI template: _Resources\\AppIcon\\appicon.svg_ for the background and _Resources\\AppIcon\\appiconfg.svg_ for the foreground.
+
+### Adaptive launcher
+
+.NET MAUI supports creating an adaptive launcher icon on Android 8.0 and higher, from the app icon. Adaptive launcher icons can display as various shapes across different device models, including circular and square. For more information about adaptive icons, see the [Android developer guide: Adaptive icons](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive).
+
+Adaptive launcher icons are composed icons, using a background layer and a foreground layer, and an optional scaling value. For more information, see the [Composed icon section](#composed-icon). If the composed icon is defined, by specifying the `ForegroundFile` attribute, an adaptive launcher icon will be generated. The following XML demonstrates defining an icon that is used as an adaptive launcher icon:
+
+```xml
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.svg" ForegroundFile="Resources\AppIcon\appiconfg.svg" ForegroundScale="0.65" Color="#512BD4" />
+</ItemGroup>
+```
+
+The `ForegroundScale` attribute can be optionally specified to change the scaling of the foreground layer being rendered over the background layer.
+
+# [iOS & macOS](#tab/ios)
+
+The app icon defined by your .NET MAUI app is used to generate an asset catalog icon set for both iOS and macOS platforms. The name of the icon set is defined in the plist file, which on iOS is located at _Platforms\\iOS\\Info.plist_. For macOS, the plist file is located at _Platforms\\MacCatalyst\\Info.plist_.
```

Comment:
Should always specify throughout this section that it's an Info.plist file, rather than a plist file. There are other plist files that can go in the iOS resources.